.github/actions/setup-env/action.yml

```diff
@@ -0,0 +1,53 @@
+name: Setup Package Environment
+description: Based on the provided package name, this action will map and set up the necessary environment variables
+inputs:
+  pkg_name:
+    description: 'The package this will setup the environment for'
+    required: true
+
+runs:
+  using: composite
+  steps:
+      - name: Setup env variables based on pkg_name
+        env:
+          PACKAGE_NAME: ${{ inputs.pkg_name }}
+        run: |
+          local WORKSPACE_PATH=''
```

Comment:
You can put a script file next to the action. Like the publish pages action. https://github.com/launchdarkly/gh-actions/blob/main/actions/publish-pages/action.yml

Reply:
Done